x-pack/test/functional/apps/canvas/embeddables/lens.ts

```diff
@@ -34,32 +34,8 @@ export default function canvasLensTest({ getService, getPageObjects }: FtrProvid
       await kibanaServer.savedObjects.cleanStandardList();
     });
 
-    describe('by-reference', () => {
-      it('adds existing lens embeddable from the visualize library', async () => {
-        await canvas.clickAddFromLibrary();
-        await dashboardAddPanel.addEmbeddable('Artistpreviouslyknownaslens', 'lens');
-        await testSubjects.existOrFail('embeddablePanelHeading-Artistpreviouslyknownaslens');
-      });
-
-      it('edits lens by-reference embeddable', async () => {
-        await dashboardPanelActions.editPanelByTitle('Artistpreviouslyknownaslens');
-        await lens.save('Artistpreviouslyknownaslens v2', false, true);
-        await testSubjects.existOrFail('embeddablePanelHeading-Artistpreviouslyknownaslensv2');
-      });
-
-      it('renders lens visualization using savedLens expression', async () => {
-        // load test workpad
-        await canvas.goToListingPage();
-        await canvas.loadFirstWorkpad('Test Workpad');
-        await header.waitUntilLoadingHasFinished();
-
-        await lens.assertLegacyMetric('Maximum of bytes', '16,788');
-      });
-    });
```

Comment:
Why were these tests deleted?

Reply:
I just moved them after the by-value tests https://github.com/elastic/kibana/pull/182535/files/67e4a6be923bc70406add027f8ec669245e3ccef#diff-6ce27bc56c9511d63d711cfb442c7124206f48e98cca5c287d67e3c12e834098R65-R89